src/convertToObject.js

```diff
@@ -5,8 +5,19 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+
+function convertToObject(stylesString) {
+  const stylesObject = {};
+
+  stylesString.split(';').forEach((style) => {
+    const [property, value] = style.split(':');
```

Comment:
When splitting the `style` string by ':', it would be safer to use `style.split(':', 2)` to ensure that only the first colon is used for splitting, in case there are multiple colons in the value.